main.py

```diff
@@ -1,10 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+from json import load
 
 
 def main() -> None:
-    pass
+    data = []
+
+    with open("players.json", "r") as file:
+        data = load(file)
+
+    for nickname, item in data.items():
+        race = item["race"]
+        guild = item["guild"]
+        skills = race["skills"]
```

Comment:
You should use get method here, because you will get a key error, and a check in 30th line will not work, for example